src/components/SocialListItem.tsx

```diff
@@ -1,42 +1,39 @@
 // Libraries
 import React from "react"
-import styled from "@emotion/styled"
+import { Flex, Box, chakra } from "@chakra-ui/react"
 
 // Components
 import Icon from "./Icon"
 
 // Styles
-const Item = styled.div`
-  display: flex;
-  width: 100%;
-  padding: 0.5rem 0;
-  align-items: center;
-`
-
-const StyledIcon = styled(Icon)`
-  padding-right: 0.75rem;
-`
-
-const ChildrenContainer = styled.div`
-  font-style: italic;
-
-  > a {
-    font-style: normal;
-  }
-`
+const StyledIcon = chakra(Icon)
```

Comment:
Lets just use the Chakra `Icon` component. We are not using our custom Icon component anymore. https://github.com/ethereum/ethereum-org-website/blob/dev/docs/chakra-migration-guide.md#update-dependencies

Reply:
Alright. Thanks, @pettinarip ! I'll make the requested changes ASAP.

Reply:
@pettinarip I have migrated the Social icons to use Chakra Icon. Please review the PR again. 

Also, @minimalsm, how do I reach the Forum page? On my end, the only files that use the `SocialListItem` component are `.md` files